app/src/main/res/layout/topic_info_fragment.xml

```diff
@@ -21,7 +21,8 @@
     <androidx.constraintlayout.widget.ConstraintLayout
       android:layout_width="match_parent"
       android:layout_height="wrap_content"
-      android:paddingBottom="24dp">
+      android:paddingTop="@dimen/topic_info_fragment_padding_top"
+      android:paddingBottom="@dimen/topic_info_fragment_padding_bottom">
 
       <org.oppia.android.app.customview.LessonThumbnailImageView
         android:id="@+id/topic_thumbnail_image_view"
```

Comment:
This has not been changed at all and if you see original code we do use this `LessonThumbnailImageView ` in all files but in `layout-land/topic_info_fragment.xml` therefore there is definitely some mistake in this PR.

You can check that easily by creating before/after screenshot of this screen on mobile-portrait, mobile-landscape, tablet-portrait, tablet-landscape.

Reply:
@rt4914 
So does that mean do we only have to delete layout file for tablet portrait?

Reply:
We cannot merge files in this issue, because the UI for all four combinations is very different. I think closing this PR is more correct.